src/viam/ur/module/ur_arm.cpp

```diff
@@ -35,6 +35,12 @@ constexpr char k_input_recipe[] = "/src/control/rtde_input_recipe.txt";
 constexpr auto k_noop_delay = std::chrono::milliseconds(2);     // 2 millisecond, 500 Hz
 constexpr auto k_estop_delay = std::chrono::milliseconds(100);  // 100 millisecond, 10 Hz
 
+// define callback function to be called by UR client library when program state changes
+void reportRobotProgramState(bool program_running) {
+    // Print the text in green so we see it better
+    VIAM_SDK_LOG(info) << "\033[1;32mUR program running: " << std::boolalpha << program_running << "\033[0m";
```

Comment:
Would you please file a ticket to 1) remove the use of color logging in this repo, and 2) ensure that we aren't permanently altering the stream state? If the boost logging stuff isn't doing its own stream state protection, then we need to.

Reply:
[RSDK-11048](https://viam.atlassian.net/browse/RSDK-11048?atlOrigin=eyJpIjoiMjliOTNlNWE3MzhiNDQ2NDgzODQxZDU3ZTc3YzViMzUiLCJwIjoiaiJ9)